src/mmw/js/src/core/modals/models.js

```diff
@@ -30,8 +30,35 @@ var ShareModel = Backbone.Model.extend({
     }
 });
 
+var AlertTypes = {
+    info: {
+        header: 'Information',
+        alertIcon: '\"fa fa-info-circle\"',
+        dismissLabel: 'Okay'
+    },
+    warn: {
+        header: 'Warning',
+        alertIcon: '\"fa fa-exclamation-triangle\"',
+        dismissLabel: 'Okay'
+    },
+    error: {
+        header: 'Error',
+        alertIcon: '\"fa fa-ban\"',
+        dismissLabel: 'Okay'
```

Comment:
Consider extracting `dismissLabel` into `AlertModal` since it has the same value in all three cases. Also, having a default there would allow us to not having to specify it in `modeling/models.js:965` below.